core/src/main/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpDocServicePlugin.java

```diff
@@ -77,12 +81,16 @@
     private static final String QUERY_PARAM = "query";
     private static final String HEADER_PARAM = "header";
 
+    // The formats defined by OpenAPI Specification
+    // (https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#dataTypes):
     @VisibleForTesting
     static final TypeSignature VOID = TypeSignature.ofBase("void");
     @VisibleForTesting
     static final TypeSignature BOOL = TypeSignature.ofBase("boolean");
+    // Not defined in the spec, but added to support char and byte types.
```

Comment:
Isn't `char` a 16-bit unsigned integer, which is not 8-bit?

Reply:
Oops~ Thanks~